tests/Unit/PointwiseFunctions/EquationsOfState/Test_Tabulated.cpp

```diff
@@ -0,0 +1,158 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <hdf5.h>
+#include <memory>
+#include <regex>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "IO/Connectivity.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/CheckH5.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/Helpers.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/ReadTabulated.hpp"
+#include "IO/H5/Version.hpp"
+#include "IO/VolumeDataFile.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "PointwiseFunctions/EquationsOfState/Tabulated.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+void test_tabulated() noexcept {
+  std::vector<double> logrho_data = {3.0239960056064277, 3.0573293389397609};
+  std::vector<double> logrho_from_file =
+      EquationsOfState::logrho_from_tabulated();
```

Comment:
Does the h5 file already exist at this point? I would expect it to be created then read and then deleted, unless you decided on a different way to test the tabulated functions.

Reply:
We don't have a good way to create it, so we decided to include one in the repo.  That said, the example file doesn't appear to have been provided with this PR.